src/styles/utils/normalize.scss

```diff
@@ -0,0 +1,17 @@
+* {
+  box-sizing: border-box;
+}
+
+html {
+  font-family: Roboto, sans-serif;
+  color: #060b35;
+}
+
```

Comment:
you neet to add (because without this property dots appear on the links)
```
ul {
  list-style: none;
}
```

Reply:
fixed